P5/Source/Guidelines/en/PH-PrimarySources.xml

```diff
@@ -11,7 +11,7 @@ $Id$
      type="div1" xml:id="PH" n="18">
    <head>Representation of Primary Sources</head>
 
-   <p>This chapter described elements for the encoding of materials where issues such as layout,
+   <p>This chapter describes elements for the encoding of materials where issues such as layout,
```

Comment:
@raffazizzi @MegJBrown something is missing in the first sentence

Reply:
Fixed, thanks!